src/Libraries/RevitNodes/Elements/InternalUtilities/ElementQueries.cs

```diff
@@ -10,6 +10,31 @@ namespace Revit.Elements.InternalUtilities
     [IsVisibleInDynamoLibrary(false)]
     public static class ElementQueries
     {
+        private static readonly Dictionary<Type, string> ClassFilterExceptions = new Dictionary<Type, string>
```

Comment:
I believe what you actually want here is a hashet<t> as your stored value is not really used right?